index.html

```diff
@@ -56,21 +56,23 @@
         <div class="modal-content">
             <div class="modal-header">
                 <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-                <h4 class="modal-title">About <strong>WA Reader</strong></h4>
+                <center><h4 class="modal-title">About <strong>WA Reader</strong></h4></center>
+                
             </div>
             <div class="modal-body text-justify">
-                <center>
-                    <img src="{{ url_for('static', filename='img/logo.png') }}" style="width: 40%">
-                </center>
+                    <center><img src="static/img/logo.png" style="width: 20%"></center>
```

Comment:
I see you only reduced the size of image from 40% to 20%. Any aesthetic reason for doing so?